packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -71,9 +76,76 @@ export type CollapsibleProps = {
 } & EnsembleWidgetProps &
   HasItemTemplate;
 
+const Panel = ({
+  scope,
+  setTemplateItems,
+  template,
+}: {
+  scope: CustomScope;
+  template: CollapsibleItem;
+  setTemplateItems: React.Dispatch<React.SetStateAction<CollapsibleItem[]>>;
+}): null => {
+  const evaluated = useEvaluate({ ...template }, { context: scope });
+  const itemAddedRef = useRef(0);
+
+  const evaluatedItem = useMemo(() => {
+    const { key, label, children } = evaluated as {
+      [key: string]: Expression<string>;
+    };
+
+    const header = isString(label)
+      ? label
+      : EnsembleRuntime.render([unwrapWidget(label)]);
+
+    const content = isString(children)
```

Comment:
The content can not be a string.

Reply:
what if the user passed the string as children of collapsible item ?
For ex.

```yaml
Collapsible:
  value: [2]
  item-template:
    data: [1, 2, 3]
    name: p
    template:
      CollapsibleItem:
        key: ${p}
        label: ${'hello' + getDateLabel(p)}
        children: hiii there ${p}
```

Reply:
Then that is invalid and an exception will be thrown.